### PR TITLE
add ncnews-10

### PR DESCRIPTION
**Pull request by wbaig2**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -314,3 +314,61 @@ describe("GET /api/articles/:article_id/comments", () => {
...
+    return request(app)
+      .post("/api/articles/500/comments")
+      .send(newComment)
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Article not found");
+      });
+  });
+
+  test("Responds with a status 400 when an article_id is provided", () => {
```
Comment (karlaferry) [inline __tests__/app.test.js:359]: Please edit your test description to make it clearer what this test does.

File: __tests__/app.test.js [hunk 2]
```diff
@@ -314,3 +314,61 @@ describe("GET /api/articles/:article_id/comments", () => {
...
+
+    return request(app)
+      .post("/api/articles/bananas/comments")
+      .send(newComment)
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Invalid article ID provided");
+      });
+  });
+
```
Comment (karlaferry) [inline __tests__/app.test.js:373]: Please add more error handling, for instance, what if the username doesn't exist in the database? Or what if the post body does not have the required properties?

**Review by karlaferry — CHANGES_REQUESTED** [review]
Almost there :)

File: __tests__/app.test.js (hunk 2 above)
Comment (HeatherEver) [reply __tests__/app.test.js:388]: there just needs to be a test for an empty body field

**Review by HeatherEver — APPROVED** [review]
just one more test to add to this endpoint
